lib/ronin/app/cli.rb

```diff
@@ -142,6 +150,26 @@ def is_redis_running?
         !`pgrep redis-server`.empty?
       end
 
+      #
+      # Determines if the Valkey server is running.
+      #
+      # @return [Boolean]
+      #   Specifies whether the `valkey-server` process is running or not.
+      #
+      def is_valkey_running?
+        !`pgrep valkey-server`.empty?
+      end
+
+      #
+      # Determines if the Valkey server is installed.
+      #
+      # @return [Boolean]
+      #   Specifies whether the `valkey-server` is installed or not.
+      #
+      def is_valkey_installed?
+        system("command -v valkey-server > /dev/null 2>&1")
```

Comment:
This `system(...)` call could be replaced by requiring `command_kit/env/path` and including `CommandKit::Env::Path` for the [`command_installed?`](https://rubydoc.info/gems/command_kit/CommandKit/Env/Path#command_installed%3F-instance_method) method, which should be faster than executing the `command` command to check if `valkey-server` is installed.